### PR TITLE
Add a release checklist

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: Release checklist
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all the important changes in `appendix/changelog.rst`
+- [ ] Set the release version and date in `appendix/changelog.rst`
```
Comment (seisman) [inline .github/ISSUE_TEMPLATE/release-checklist.md:11]:
```suggestion
- [ ] Set the release version and date in `appendix/changelog.rst`
- [ ] Set the release version `SACDOCVERSION` in `source/SAC_style.sty`
- [ ] Set the release version in `source/conf.py`
- [ ] Add the release information to `source/version.rst`
- [ ] Upload the PDF to the release
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- [ ] Set the release version `SACDOCVERSION` in `source/SAC_style.sty`
+- [ ] Set the release version in `source/conf.py`
+- [ ] Add the release information to `source/version.rst`
+- [ ] Upload the PDF to the release
```
Comment (seisman) [inline .github/ISSUE_TEMPLATE/release-checklist.md:15]:
```suggestion
- [ ] Upload the PDF to the release
- [ ] Bump the version number to "x.x-dev" in `source/conf.py` and `source/SAC_style.sty`
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Bump the version number to "x.x-dev" in `source/conf.py` and `source/SAC_style.sty`
```
Comment (seisman) [inline .github/ISSUE_TEMPLATE/release-checklist.md]:
```suggestion
- [ ] Bump the version number to "X.Y-dev" in `source/conf.py` and `source/SAC_style.sty`
- [ ] Bump the SAC version number in `source/SAC_style.sty` if necessary
```

Comment (core-man) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Add bumping SAC version?

Comment (seisman) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Please see the updated suggestion above.

**Review by seisman — APPROVED** [review]

**Comment by core-man** [comment]
could i merge to master?

**Comment by seisman** [comment]
Yes, all future PRs will be merged into master.

**Pull request merged**